components/cli/cmd/cellery/describe.go

```diff
@@ -48,7 +50,9 @@ func newDescribeCommand() *cobra.Command {
 			return nil
 		},
 		Run: func(cmd *cobra.Command, args []string) {
-			commands.RunDescribe(args[0])
+			if err := image.RunDescribe(cli, args[0]); err != nil {
```

Comment:
Can't we use RunE everywhere?

Reply:
If we return the error to the main function using RunE, when printing the error message, the usage will also get printed.

Reply:
Did you check `SilenceUsage`?

https://godoc.org/github.com/spf13/cobra#Command